modules/osdk-scorecard-config.adoc

```diff
@@ -0,0 +1,185 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-scorecard.adoc
+
+[id="osdk-scorecard-config_{context}"]
+= Scorecard configuration
+
+The scorecard tool uses a configuration file that allows configuring internal
+plug-ins as well as a few global configuration options.
+
+[id="osdk-scorecard-config-file_{context}"]
+== Configuration file
+
+The scorecard tool's configuration file default location is the
```

Comment:
only the yaml file format of the scorecard configuration is accepted, toml/json support was removed from this version of scorecard.

Reply:
Suggest "The default location for the scorecard tool's configuration file is"